openff/toolkit/topology/molecule.py

```diff
@@ -4246,50 +4246,56 @@ def _add_conformer(self, coordinates):
         index: int
             The index of this conformer
         """
-        new_conf = unit.Quantity(
-            np.zeros(shape=(self.n_atoms, 3), dtype=float), unit.angstrom
-        )
```

Comment:
I found this confusing - this `new_conf` read to me like the conformer passed through as an argument, but it's just a temporary variable used for  input validation. `coordinates` is the argument.